test/openshift/e2e/operator_expectations.go

```diff
@@ -8,6 +8,7 @@ import (
 	"fmt"
 
 	"github.com/golang/glog"
+	mapiv1alpha1 "github.com/openshift/cluster-api/pkg/apis/machine/v1beta1"
```

Comment:
mapiv1beta1

Reply:
other than this, lgtm if @bison @ingvagabund can take another look and agree, once this is addressed let's squash a bit

Reply:
Fixed in https://github.com/openshift/kubernetes-autoscaler/pull/29/commits/6434ec24d58803f2e6535d7e2ee8edc4c83a9d18

Reply:
Yeah, agree. Looks good. Just please squash the commits a bit e.g. rename cluster-api imports in one commit instead of two.

Reply:
Squashed in https://github.com/openshift/kubernetes-autoscaler/commit/b2ad7e615c42d328ba4929ce0845ce9e94f9ab9e